lib/voxpupuli/release/rake_tasks.rb

```diff
@@ -47,7 +47,8 @@
   #
   # ## 2016-11-20 Release 4.0.2
   # ## [v4.0.3-rc0](https://github.com/voxpupuli/puppet-r10k/tree/v4.0.3-rc0) (2016-12-13)
-  if File.readlines('CHANGELOG.md').grep( /^(#.+[Rr]eleas.+#{Regexp.escape(v)}|## \[v#{Regexp.escape(v)}\])/ ).size == 0
+  # ## [2.1.0](https://github.com/opus-codium/puppet-odoo/tree/2.1.0) (2021-12-02)
+  if File.readlines('CHANGELOG.md').grep( /^(#.+[Rr]eleas.+#{Regexp.escape(v)}|## \[v?#{Regexp.escape(v)}\])/ ).size == 0
```

Comment:
Unrelated, I wonder if this also works:
```suggestion
  if File.readlines('CHANGELOG.md').grep( /^(#.+[Rr]eleas.+#{Regexp.escape(v)}|## \[v?#{Regexp.escape(v)}\])/ ).none?
```
That would use https://ruby-doc.org/core-3.0.3/Enumerable.html#method-i-none-3F

Reply:
[`Array#empty?`](https://ruby-doc.org/core-3.0.3/Array.html#method-i-empty-3F) can make even more sense:
```suggestion
  if File.readlines('CHANGELOG.md').grep( /^(#.+[Rr]eleas.+#{Regexp.escape(v)}|## \[v?#{Regexp.escape(v)}\])/ ).empty?
```

Reply:
@smortex I like yourbcode suggestion. should we merge that and add it to the release PR thats open?

Reply:
Commit amended to include this change!

Reply:
Lovely. I always enjoy when code is readable and `empty?` is much more expressive in its intent than `size == 0`.